### PR TITLE
Update main's tools

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

**Comment by natefoo** [comment]
This contains an update of the Main version as well, but since @davebx is out for the day I'm going to merge and then just change it back.

File: files/galaxy/usegalaxy.org/var/shed_tool_conf.xml [hunk 1]
```diff
@@ -6858,4 +6857,256 @@
     </tool>
 </section>
 
+<section id="ngs:_gemini" name="NGS: Gemini" version="">
+  <tool file="toolshed.g2.bx.psu.edu/repos/iuc/gemini_de_novo/285f519dab87/gemini_de_novo/gemini_de_novo.xml" guid="toolshed.g2.bx.psu.edu/repos/iuc/gemini_de_novo/gemini_de_novo/0.18.1.0">
```
Comment (martenson) [inline files/galaxy/usegalaxy.org/var/shed_tool_conf.xml]: this could benefit from `labels="new"`
@davebx

**Comment by martenson** [comment]
this could benefit from `labels="new"`

File: files/galaxy/usegalaxy.org/var/integrated_tool_panel.xml [hunk 2]
```diff
@@ -918,4 +918,29 @@
     <tool id="echo_main_w2_handler2" />
     <tool id="echo_main_stampede" />
     <tool id="echo_main_cluster" />
+    <section id="ngs:_gemini" name="NGS: Gemini" version="">
```
Comment (martenson) [inline files/galaxy/usegalaxy.org/var/integrated_tool_panel.xml]: this section needs to be elsewhere in the tool panel

**Pull request merged**